pkg/apis/contrail/v1alpha1/swiftproxy_types.go

```diff
@@ -29,17 +29,12 @@ type SwiftProxyConfiguration struct {
 	RingConfigMapName     string       `json:"ringConfigMapName,omitempty"`
 	Containers            []*Container `json:"containers,omitempty"`
 	SwiftServiceType      serviceType  `json:"swiftServiceType,omitempty"`
+	// MetalLB pool name that should be used by Service
+	IPPoolName string `json:"ipPoolName,omitempty"`
```

Comment:
Is it too specific? That we create metalb field in this api? Maybe we should create some sub structure like
```
type Service struct {
type 
annotations
}
```
That might be more general not limited to metallb

Reply:
so you want to pass annotations specified by the user?

Reply:
done